### ISSUE TITLE
Weird behavior with special comments inside style block.

### ISSUE THREAD

**Issue by wnr** [jakubpawlowicz/clean-css#809]
I run the following example with the command `node_modules/.bin/cleancss --s0 css/test.css`. 

Input:

``` css
.foo {
  color: red;
  /*!*/







  /*!*/
}
```

Output:

``` css
.foo{color:red







  }
```

The expected result is `.foo{color:red}`. It seems that all other combinations of the content of the style rule results in the expected output.

**Comment by jakubpawlowicz** [comment]
Interesting. Let's have it fixed.

**Comment by jakubpawlowicz** [comment]
Thanks to the new tokenizer this is no longer an issue. Due to be released in 4.0. Thanks @wnr!
